sagemaker-python-sdk/tensorflow_script_mode_using_shell_commands/tensorflow_script_mode_using_shell_commands.ipynb

```diff
@@ -0,0 +1,351 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# TensorFlow Script Mode - Using Shell commands\n",
+    "\n",
+    "Starting from TensorFlow version 1.11, you can use shell commands as\n",
+    "your training entry point. Shell scripts are useful for many use cases including:\n",
+    "\n",
+    "- Invoke Python scripts with specific parameters\n",
+    "- Configure framework dependencies\n",
+    "- Train using different computer languages\n",
+    "\n",
+    "For this example, we use [a Keras implementation of the Deep Dream algorithm](https://github.com/keras-team/keras/blob/2.2.4/examples/deep_dream.py). We can use the same technique for other scripts or repositories including [TensorFlow Model Zoo](https://github.com/tensorflow/models) and [TensorFlow benchmark scripts](https://github.com/tensorflow/benchmarks/tree/master/scripts/tf_cnn_benchmarks)."
```

Comment:
It is open sourced https://github.com/keras-team/keras/blob/2.2.4/LICENSE

Reply:
I ask because generally images are described as being under public domain or a creative commons license - open source seems to be applied to text/code

Reply:
I see, I got confused: You are referring to the image not the code. I actually had to change to an image under creative commons. Thanks!